test/studies/shootout/submitted/knucleotide-submitted.graph

```diff
@@ -1,5 +1,5 @@
 perfkeys: real, real
-graphkeys: release, submitted
-files: knucleotide.dat, knucleotide-submitted.dat
+graphkeys: release, submitted, knucleotide4
```

Comment:
Should we give this a more descriptive name? I don't know enough about this test to say what's the distinguishing feature of this.

Reply:
The submitted name is knucleotide4, but would you prefer something like `knucleotide-hash` instead? (the hash is the distinguishing feature)

Reply:
If that's what sets this version apart from `release` and `submitted`, sure.

Reply:
Alright, should be good to go